devices.js

```diff
@@ -2485,6 +2485,33 @@ const devices = [
         fromZigbee: [fz.diyruz_rspm],
         toZigbee: [tz.on_off],
     },
+    {
+        zigbeeModel: ['DIYRuZ_FreePad'],
+        model: 'DIYRuZ_FreePad',
+        vendor: 'DIYRuZ',
+        description: '[DiY 8/12/20 button keypad](http://modkam.ru/?p=1114)',
+        supports: 'single, double, triple, quadruple, many, hold/release',
+        fromZigbee: [fz.diyruz_freepad_clicks, fz.battery],
+        toZigbee: [tz.diyruz_freepad_on_off_config],
+        meta: {configureKey: 1},
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(1);
+            await bind(endpoint, coordinatorEndpoint, ['genPowerCfg']);
+            await configureReporting.batteryPercentageRemaining(endpoint);
+            await configureReporting.batteryVoltage(endpoint);
+            device.endpoints.forEach(async (ep) => {
+                await bind(ep, coordinatorEndpoint, ['genMultistateInput']);
+            });
+        },
+        endpoint: (device) => {
+            return {
+                'btn_1': 1, 'btn_2': 2, 'btn_3': 3, 'btn_4': 4, 'btn_5': 5,
```

Comment:
All endpoint names need to be defined here: https://github.com/Koenkk/zigbee2mqtt/blob/master/lib/util/utils.js#L30.

Would it make sense to use `ep1`, `ep2`, etc.?

Reply:
It have more sense to rename `button_1`, `button_2`, since it always used for action name generation. button_1_click has more sense than ep_1_click

Reply:
True, in that case also make a pr for https://github.com/Koenkk/zigbee2mqtt/blob/master/lib/util/utils.js#L33 to add all buttons.

Reply:
Done @Koenkk 
https://github.com/Koenkk/zigbee2mqtt/pull/3552